tests/helics/shared_library/evilInputTests.cpp

```diff
@@ -2615,6 +2677,20 @@ TEST(evil_input_test, helicsInputSetInfo)
     EXPECT_NE(err.error_code, 0);
 }
 
+TEST(evil_input_test, helicsInputSetMinimumChange)
+{
+    //void helicsInputSetMinimumChange(helics_input inp, double tolerance, helics_error* err);
+    char rdata[256];
+    helics_input evil_input = reinterpret_cast<helics_input>(rdata);
+    auto err = helicsErrorInitialize();
+    err.error_code = 45;
+    helicsInputSetMinimumChange(nullptr, 12.0, &err);
```

Comment:
What are lines 2686-2688 testing? Right now it looks like it could be testing what happens when passing in a nullptr, or an error object with the error_code already set (though helicsErrorClear on 2688 looks like it would ignore whatever happens).